cmd/bblfsh-sdk/cmd/build.go

```diff
@@ -32,6 +32,8 @@ func (c *BuildCommand) Execute(args []string) error {
 
 const TestCommandDescription = "tests the driver using fixtures"
 
+// TestCommand will compare gold annotations in driver ./fixtures to the
```

Comment:
s/gold/old

Reply:
Hm, I think usage of the word "gold" is common when referring to a "universal source of truth".
Was meant in that sense only. Otherwise need to clarify what are old and what are new.

If you have an exact wording - please post here (btw I find using a [new github suggestions feature](https://twitter.com/github/status/1053360435074949120?lang=en) to be faster for such cases)

Reply:
Tried to make it a bit more clear in b1de985